commands/services/sqs.py

```
@@ -0,0 +1,844 @@
+                        'Attributes': msg.get('Attributes', {}),
+                        'MessageAttributes': msg.get('MessageAttributes', {}),
+                        'MD5OfBody': msg.get('MD5OfBody'),
+                        'MD5OfMessageAttributes': msg.get('MD5OfMessageAttributes')
+                    }
+
+                    try:
+                        processed_msg['ParsedBody'] = json.loads(msg.get('Body', '{}'))
+                    except (json.JSONDecodeError, TypeError):
+                        processed_msg['ParsedBody'] = None
+
+                    messages_buffer.append(processed_msg)
+                    receipt_handles_buffer.append(msg['ReceiptHandle'])
+
+                    with stats['lock']:
+                        stats['received'] += 1
+
+                    if len(messages_buffer) >= max_messages_per_file:
+                        with stats['lock']:
+                            stats['file_counter'] += 1
+                            file_num = stats['file_counter']
+
+                        output_file = base_dir / f"messages_{file_num:04d}.jsonl"
+                        content = ''.join(
+                            json.dumps({k: v for k, v in msg.items() if k != 'ReceiptHandle'}, default=str) + '\n'
+                            for msg in messages_buffer
+                        )
+                        output_file.write_text(content)
+
+                        with stats['lock']:
+                            stats['written'] += len(messages_buffer)
+
+                        if delete_after_write:
+                            deleted = self.delete_message_batch(queue_url, receipt_handles_buffer)
+                            with stats['lock']:
+                                stats['deleted'] += deleted
+
+                        messages_buffer = []
+                        receipt_handles_buffer = []
+
+            except Exception:
+                if stop_event.is_set():
+                    break
```

Should probably log something here and not just swallow the exception?